activejob/lib/active_job/serializers/base_serializer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module ActiveJob
+  module Serializers
+    class BaseSerializer
+      class << self
+        def serialize?(argument)
+          argument.is_a?(klass)
+        end
```

Comment:
This should implement `klass`, `deserialize?`, `serialize` and `deserialize` and raise a `NotImplementedError`